### PR TITLE
Update cache s3 multi region

### PR DESCRIPTION
**Pull request by DanRoscigno**
the cache system was changed, updating pages @e-mars

### CODE REVIEW

**Review by e-mars — CHANGES_REQUESTED** [review]
Maybe in the note, add a link to https://clickhouse.com/docs/en/operations/storing-data/#using-local-cache since there was a change in 22.8 in cache configuration?

**Comment by DanRoscigno** [comment]
> Maybe in the note, add a link to https://clickhouse.com/docs/en/operations/storing-data/#using-local-cache since there was a change in 22.8 in cache configuration?

Which note?

**Review by e-mars — APPROVED** [review]
discussed note in slack.

File: docs/en/integrations/data-ingestion/s3/configuring-s3-for-clickhouse-use.md [hunk 1]
```diff
@@ -175,8 +178,7 @@ vim /etc/clickhouse-server/config.d/storage_config.xml
 ```
 
 :::note
-The tag `<s3_disk>` within the `<disks>` tag is an arbitrary label. This can be set to something else but the same label must be used in the `<disk>` tab under the `<policies>` tab to reference the disk.
-The `<metadata_path>` and `<cache_path>` are recommended to also include the name in the path to be able to identify the locations on disk.
+The tags `s3_disk` and `s3_cache` within the `<disks>` tag are arbitrary labels. These can be set to something else but the same label must be used in the `<disk>` tab under the `<policies>` tab to reference the disk.
 The `<S3_main>` tag is also arbitrary and is the name of the policy which will be used as the identifier storage target when creating resources in ClickHouse.
```
Comment (DanRoscigno) [inline docs/en/integrations/data-ingestion/s3/configuring-s3-for-clickhouse-use.md:183]:
```suggestion

The configuration shown above is for ClickHouse version 22.8 or higher, if you are using an older version please see the [storing data](/docs/en/operations/storing-data.md/#using-local-cache) docs.

```

**Pull request merged**